tests/unit/test_bucket.py

```diff
@@ -350,6 +350,7 @@ def _make_bucket():
 
         return mock.create_autospec(Bucket, instance=True)
 
+    @pytest.mark.skip(reason="Unspecified PAP is changing to inherited")
```

Comment:
We shouldn't need to disable the unit test just now -- it still passes, and skipping it causes a coverage failure.

Reply:
Ah ok, that makes sense, thanks!